### ISSUE TITLE
feature request: arm64 support

### ISSUE THREAD

**Issue by GreatHaop** [yuk7/ArchWSL#306]
**IMPORTANT**
Please read [README](https://github.com/yuk7/ArchWSL/blob/master/README.md) and [Docs](https://git.io/arch-doc) before creating the issue.

**Please fill out the below information:**
**Describe the issue**
ArchWSL currently doesn't support windows on arm64

**Comment by hydranix** [comment]
Arm support would likely require an arm kernel and archlinux only supports x86 officially.

Take a look at ArchLinuxArm for unofficial archlinux arm support. It may be possible to run alarm on WSL

**Comment by Xazax-hun** [comment]
> Arm support would likely require an arm kernel and archlinux only supports x86 officially.

I think WSL is using Microsoft's kernel, so the distros only provide the user space. And Microsoft does support WSL on arm64, and there is a Ubuntu variant for WSL on arm64. I might be missing something, but I don't see this as a blocker.

**Comment by hydranix** [comment]
Yeah, which requires an archlinux userspace compiled for arm CPU. Which ArchLinux does not support. They only provide packages compiled for x86.

ArchLinuxArm however does provide an arm userspace.

The kernel will be the microsoft kernel or one based on its configs regardless of whatever userspace is used.

**Comment by LNKLEO** [comment]
https://cordx56.hatenablog.com/entry/2020/11/22/162253

try this. i was using this with my surface pro x before.
